### PR TITLE
l10n: sync ar de_DE es_ES fr_FR nb_NO nl pt_BR translations

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

Update the latest translations from https://weblate.securedrop.club/

## Testing

These are new translations files (covered by a generic apparmor inclusion) and updates that can be verified if you speak the language by switching to the corresponding language using the menu. Ensuring there is no breakage because of placeholders incorrectly copied over can be done with:

<pre>
$ PAGE_LAYOUT_LOCALES='ar,de_DE,es_ES,fr_FR,nb_NO,nl,pt_BR' pytest -v --page-layout tests/pages-layout
...
================================================ 357 passed, 3 warnings in 1996.80 seconds =================================================
</pre>

## Deployment

The new files will be added when the package is updated. There is no action required from the sysadmin point of view.

### If you made changes to the app code:

- [x] Unit and functional tests pass on the development VM

### CODE REVIEW

**Comment by dachary** [comment]
@redshiftzero although we could wait until all the translations are fully reviewed, I would feel more confortable if this large update, including new files and lots of changes are merged earlier. I will then wait until the last minute to submit another pull requests, most likely with minor changes from the reviewers.

**Comment by dachary** [comment]
Unrelated failure https://circleci.com/gh/freedomofpress/securedrop/4291?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link re-pushing to verify this is a transient error.
<pre>
    RUNNING HANDLER [tor-hidden-services : restart tor (simple)] *******************
    Wednesday 29 November 2017  15:42:10 +0000 (0:00:00.035)       0:04:52.090 ****
    fatal: [app-staging]: FAILED! => {"changed": false, "failed": true, "msg": " * Stopping tor daemon...\n   ...done.\n * Starting tor daemon...\n   ...fail!\n"}
    changed: [mon-staging]
</pre>

**Comment by msheiny** [comment]
> Unrelated failure https://circleci.com/gh/freedomofpress/securedrop/4291?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link re-pushing to verify this is a transient error.

I've been seeing that issue randomnly against installs I have in CI that are supposed to trigger off the tor nightly (unstable) repo -- I thought it was an issue strictly with tor nightly and didnt realize it was also creeping into our CI. Suspect we have to add some better logic here -- maybe tor should be restarted a few times until it comes up successfully.

**Comment by dachary** [comment]
@redshiftzero if you agree to include pt_BR in 0.5, I'll make an issue to add the corresponding flag which is still missing.

**Comment by redshiftzero** [comment]
ha you read my mind - just filed #2631

File: securedrop/translations/ar/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -311,45 +352,45 @@ msgstr "حذف"
...
+#: journalist_templates/admin_add_user.html:24
 #: journalist_templates/edit_account.html:17
 msgid "Is Administrator"
 msgstr "هو المسؤول"
 
-#: journalist_templates/admin_add_user.html:18
+#: journalist_templates/admin_add_user.html:32
 msgid "Is using a YubiKey [HOTP]"
-msgstr "هو استعمال [YubiKey [HOTP"
+msgstr "يستخدم (تستخدم) [YubiKey [HOTP"
```
Comment (redshiftzero) [inline securedrop/translations/ar/LC_MESSAGES/messages.po]: Formatting issues (to fix in Weblate)

File: securedrop/translations/ar/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -18,199 +18,237 @@ msgstr ""
...
 
-#: journalist.py:655 journalist.py:784
-msgid "No collections selected for deletion."
-msgstr "لم يتم تحديد مجموعات للحذف."
+#: journalist_app/forms.py:28
+msgid ""
+"Field must be at least {min_chars} characters long but only got {num_chars}."
+msgstr ""
+"يجب أن يتضمن هذا الحقل أو هذه الخانة 40 حرفاً كحدّ أدنى، ولكنه عوضاً عن ذلك "
+"مكوّن من {num_chars}."
```
Comment (redshiftzero) [inline securedrop/translations/ar/LC_MESSAGES/messages.po]: `{min_chars}` missing

**Pull request merged**